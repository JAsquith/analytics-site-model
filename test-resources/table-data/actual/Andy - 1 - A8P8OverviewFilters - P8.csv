Name,Filter Value,P8
Armed Forces,Y,-0.141
Data Set 4 >,,-1.616
Difference >,,1.475
SEN,A,0.563
Data Set 4 >,,0.594
Difference >,,-0.031
EAL,Y,0.075
Data Set 4 >,,0.533
Difference >,,-0.458
KS2 Banding,Middle,0.587
Data Set 4 >,,1.071
Difference >,,-0.484
KS1 - 2 Progress?,N,0.485
Data Set 4 >,,1.010
Difference >,,-0.525
Term of Birth,2,-0.309
Data Set 4 >,,0.384
Difference >,,-0.693
Term of Birth,3,0.090
Data Set 4 >,,1.053
Difference >,,-0.962
Attendance Group,1. Below 85%,0.665
Data Set 4 >,,1.765
Difference >,,-1.100
Attendance Group,4. Above 95%,-0.346
Data Set 4 >,,0.487
Difference >,,-0.833
Gender,Boys,-0.145
Data Set 4 >,,0.708
Difference >,,-0.854
SEN,N,-0.310
Data Set 4 >,,0.591
Difference >,,-0.901
SEN Provision,N,-0.310
Data Set 4 >,,0.591
Difference >,,-0.901
Ethnic Code,WENG,-0.366
Data Set 4 >,,0.625
Difference >,,-0.991
FSM Ever 6,N,-0.366
Data Set 4 >,,0.625
Difference >,,-0.991
Gifted Talented,N,-0.366
Data Set 4 >,,0.625
Difference >,,-0.991
Looked After,Yay,-0.366
Data Set 4 >,,0.625
Difference >,,-0.991
Pupil Premium,Not PP,-0.366
Data Set 4 >,,0.625
Difference >,,-0.991
Traveller,N,-0.366
Data Set 4 >,,0.625
Difference >,,-0.991
EAL,N,-0.394
Data Set 4 >,,0.631
Difference >,,-1.025
KS1 - 2 Progress?,Y,-0.440
Data Set 4 >,,0.591
Difference >,,-1.031
Attendance Group,3. 90% - 95%,-0.371
Data Set 4 >,,0.639
Difference >,,-1.010
Gender,Girls,-0.647
Data Set 4 >,,0.514
Difference >,,-1.161
Attendance Group,2. 85% - 90%,-0.539
Data Set 4 >,,0.812
Difference >,,-1.351
Armed Forces,N,-0.371
Data Set 4 >,,0.672
Difference >,,-1.043
Term of Birth,1,-0.533
Data Set 4 >,,0.624
Difference >,,-1.157
SEN Provision,K,-0.621
Data Set 4 >,,0.779
Difference >,,-1.401
KS2 Banding,Upper/High,-1.241
Data Set 4 >,,0.262
Difference >,,-1.503
SEN,P,-1.569
Data Set 4 >,,0.928
Difference >,,-2.496
KS2 Banding,Lower,2.276
Data Set 4 >,,
Difference >,,
Summary,,-0.366
Data Set 4 >,,0.625
Difference >,,-0.991