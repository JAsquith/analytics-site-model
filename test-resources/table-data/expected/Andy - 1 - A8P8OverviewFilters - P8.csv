Name,Filter Value,P8
KS1 - 2 Progress?,N,-0.668
KS4 Targets >,,0.094
Difference >,,-0.762
Term of Birth,3,-0.668
KS4 Targets >,,0.094
Difference >,,-0.762
Attendance Group,3. 90% - 95%,-1.756
KS4 Targets >,,-0.656
Difference >,,-1.100
KS2 Banding,Middle,1.459
KS4 Targets >,,2.081
Difference >,,-0.622
Term of Birth,2,1.459
KS4 Targets >,,2.081
Difference >,,-0.622
Gender,Girls,-1.618
KS4 Targets >,,-0.239
Difference >,,-1.379
Armed Forces,N,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
EAL,N,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
Ethnic Code,WENG,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
FSM Ever 6,N,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
Gifted Talented,N,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
Looked After,Yay,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
Pupil Premium,Not PP,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
SEN,N,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
SEN Provision,N,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
Traveller,N,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283
KS2 Banding,Upper/High,-1.180
KS4 Targets >,,0.214
Difference >,,-1.394
Attendance Group,4. Above 95%,-0.015
KS4 Targets >,,1.180
Difference >,,-1.195
KS1 - 2 Progress?,Y,-0.825
KS4 Targets >,,0.545
Difference >,,-1.370
Gender,Boys,-0.191
KS4 Targets >,,1.021
Difference >,,-1.212
Term of Birth,1,-1.282
KS4 Targets >,,0.238
Difference >,,-1.520
Attendance Group,2. 85% - 90%,-1.031
KS4 Targets >,,0.569
Difference >,,-1.600
Summary,,-0.803
KS4 Targets >,,0.481
Difference >,,-1.283